### ISSUE TITLE
How to change the settings of the chart opened in a new window?

### ISSUE THREAD

**Issue by saklanmazozgur** [swharden/ScottPlot#669]
[I am doing an application that prints time-dependent data on a graph. It writes the data on the chart properly over time. But when I open the graph on the new screen, y axis does not show Datetime value in "HH.mm.ss" format. It shows the double value.

Also, when I want to open the graphic on a new screen, I don't want the newly opened graphic to be right clickable again.

How can I change the settings of the graphic copy opened in a new window? How can I apply time stamping?

How to change the settings of the chart opened in a new window?

Also, can I change the title of the newly opened window?

New Window Code

`new WpfPlotViewer(Global.AxisX_Graphic.plt.Copy()).Show();`

Normally Graphic

![Zamanlı](https://user-images.githubusercontent.com/52654922/102748708-bf9d7700-4373-11eb-9fb9-56ed8c95c4df.PNG)

Open New Window Graphic 

![Zamansız](https://user-images.githubusercontent.com/52654922/102748713-c3c99480-4373-11eb-8849-69b597ca5ab6.PNG)

**Comment by saklanmazozgur** [comment]
The y-axis was displayed properly when I typed `new WpfPlotViewer(Global.AxisX_Graphic.plt.Cop).Show();` instead of `new WpfPlotViewer(Global.AxisX_Graphic.plt.Copy()).Show();.`

**Comment by saklanmazozgur** [comment]
I fixed problem. 

`var AxisX_Copy = Global.AxisX_Graphic.plt.Copy();`
`AxisX_Copy.Ticks(dateTimeX: true, dateTimeFormatStringX: ("HH.mm.ss"));`
`wpfPlotViewer = new WpfPlotViewer(AxisX_Copy);`
`wpfPlotViewer.Title = "X Axis";`
`wpfPlotViewer.Show();`
`wpfPlotViewer.wpfPlot1.ContextMenu.Items.Clear();`

**Comment by saklanmazozgur** [comment]
I fixed all problems. Respectively;

First, I copied my chart as AxisX_Copy.

`var AxisX_Copy = Global.AxisX_Graphic.plt.Copy();`

I also applied the settings I applied to the original chart to the copy.

`AxisX_Copy.Ticks(dateTimeX: true, dateTimeFormatStringX: ("HH.mm.ss"));`


then I created a new plotViewer.

`wpfPlotViewer = new WpfPlotViewer(AxisX_Copy);`

 I changed the title of the new page as follows.

`wpfPlotViewer.Title = "X Axis";`

I cleared the chart's menu in the newly opened window.

`wpfPlotViewer.Show();`
`wpfPlotViewer.wpfPlot1.ContextMenu.Items.Clear();`
